tests/lib/rules/self-closing-comp.js

```diff
@@ -107,73 +107,84 @@ ruleTester.run('self-closing-comp', rule, {
   invalid: [
     {
       code: 'var contentContainer = <div className="content"></div>;',
+      output: 'var contentContainer = <div className="content"/>;',
```

Comment:
Good catch @tleunen, I have now tested this rule with `jsx-space-before-closing` which is also fixable, and both of these fixes are working together to fix all warnings when using my initial commit. Adding a white space may be beyond the scope of this rule, as that is better left to the fix implemented in `jsx-space-before-closing`.

If you and @ljharb agree after your own testing that the fix for `jsx-space-before-closing` works well with this fix, then I think f6cc36a should be reverted.

Also from our discussion I noticed that the [jsx-space-before-closing documentation](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-space-before-closing.md) is missing the **Fixable** notice, I can add that to this PR if you like.